source/frontend/src/features/account/Login.tsx

```diff
@@ -52,49 +52,51 @@ const Login = () => {
 
   return (
     <LoginStyled className="login" fluid={true}>
-      <Container className="unauth" fluid={true}>
-        <Row>
-          <Col xs={1} />
-          <Col xs={1} md={3}>
-            <Logo withText={false} />
-          </Col>
-          <Col xs={16} md={6} className="logo-title">
-            <h1>{tenant.login.title}</h1>
-          </Col>
-        </Row>
+      <VertialAlign>
```

Comment:
This page was previously relying on static padding to center the login component, which didn't scale to other screen sizes.